advocacy_docs/partner_docs/ThalesCipherTrustManager/03-SolutionSummary.mdx

```diff
@@ -1,13 +1,13 @@
 ---
-title: 'Solution Summary'
+title: 'Solution summary'
 description: 'Explanation of the solution and its purpose'
 ---
 
-Thales CipherTrust Manager offers an industry-leading enterprise key management solution to help organizations centrally manage encryption keys, provide access control, and configure security policies. Thales CipherTrust Manager can be used in conjunction with EDB Postgres Advanced Server versions 15.2 and above and EDB Postgres Extended Server version 15.2 and above, Transparent Data Encryption (TDE) capabilities to further secure your sensitive data at-rest.
+Thales CipherTrust Manager offers an industry-leading, enterprise key-management solution to help organizations centrally manage encryption keys, provide access control, and configure security policies. You can use Thales CipherTrust Manager with EDB Postgres Advanced Server version 15.2 and later and EDB Postgres Extended Server version 15.2 and later transparent data encryption (TDE) capabilities to further secure your sensitive data at rest.
 
-Thales CipherTrust Manager manages key lifecycle tasks such as generation, rotation, destruction, import and export, and provides role-based access control to keys and policies. Thales CipherTrust Manager can be utilized in both virtual and physical appliances that integrate with FIPS 140-2 compliant third-party Hardware Security Modules (HSM) to securely store keys.  Utilizing EDB Postgres Advanced Server’s or EDB Postgres Extended Server's TDE capabilities along with Thales CipherTrust Manager for key management allows sensitive data to be protected wherever it may reside.
+Thales CipherTrust Manager manages key lifecycle tasks, such as generation, rotation, destruction, and import and export, and provides role-based access control to keys and policies. You can use Thales CipherTrust Manager in both virtual and physical appliances that integrate with FIPS 140-2-compliant third-party hardware security modules (HSM) to securely store keys. Using EDB Postgres Advanced Server’s or EDB Postgres Extended Server's TDE capabilities along with Thales CipherTrust Manager for key management allows you to protect sensitive data wherever it resides.
```

Comment:
@ebgitelman Rather than use the possessive form of those long product names, would it be clearer if they just read "EDB Postgres Advanced Server or EDB Postgres Extended Server TDE capabilities"? Just something to consider.

Reply:
Sure thing